src/oneTrust/endpoints/getAssessment.ts

```diff
@@ -236,7 +213,19 @@ export const OneTrustAssessmentResponses = t.array(
     }),
     t.type({
       /** The controlResponse */
-      controlResponse: t.union([t.string, t.null]),
+      controlResponse: t.union([
+        t.string,
+        t.type({
+          controlId: t.string,
+          controlIdentifier: t.string,
+          controlName: t.string,
+          frameworkId: t.string,
+          frameworkName: t.string,
+          categoryId: t.string,
+          categoryName: t.string,
+        }),
```

Comment:
do we have some t.object type of sorts? if so, maybe that'd be better

Reply:
also, crazy that this can be a string *and*. an object lol

Reply:
Doesn't look like it from the io-ts docs. But it also looks like maybe [we should be using `struct` instead of `type`](https://gcanti.github.io/io-ts/modules/Codec.ts.html#type). Tbh I don't really get codecs and these docs are not my fave.